### PR TITLE
Set up math magicians

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
- Set up a React app.
- Use React documentation.

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -7,4 +7,4 @@
     "csstree/validator": true
   },
   "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/*.js", "**/*.jsx"]
-}
\ No newline at end of file
+}
```
Comment (D4nielJ) [inline .stylelintrc.json:10]: - [x] I think it's better if we use the README file from [Microverse Template](https://github.com/microverseinc/readme-template). Although the one from CRA is not incorrect, it does seem generic and doesn't document the project in a very professional way. I do recommend as well including the scripts from this document in the other.

[OPTIONAL] Make sure that when you open a pull request, all of the changes required from Microverse are included in this Pull Request, this facilitates the review process as Code Reviewers are able to comment exactly on the pieces of code that might need a change.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @Lucas-Masaba, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: 
- CRA has been used to initialize the app :rocket: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .stylelintrc.json (hunk 1 above)
Comment (Lucas-Masaba) [reply .stylelintrc.json:10]: Thank you for your observation. I have indeed updated my README according to your preferences

Comment (Lucas-Masaba) [reply .stylelintrc.json:10]: I also did the [OPTIONAL] requirement. I thank you again

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Lucas-Masaba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**